src/middlewared/middlewared/plugins/smb.py

```diff
@@ -905,6 +907,7 @@ async def do_update(self, id, data):
         new['vuid'] = await self.generate_vuid(new['timemachine'], new['vuid'])
         await self.clean(new, 'sharingsmb_update', verrors, id=id)
         await self.validate(new, 'sharingsmb_update', verrors, old=old)
+        await self.legacy_afp_check(new, 'sharingsmb_verrors', verrors)
```

Comment:
_update?

Reply:
The number of typos I make increases exponentially as the number of lines in diff decreases :(